<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisassemblerViewPage</class>
 <widget class="QWidget" name="DisassemblerViewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabMain">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDisassembly">
      <attribute name="title">
       <string>Disassembly</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="DisassemblerView" name="disassemblyView"/>
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabFunctions">
           <attribute name="icon">
            <iconset resource="../../../resources.qrc">
             <normaloff>:/format_icons/res/method.png</normaloff>:/format_icons/res/method.png</iconset>
           </attribute>
           <attribute name="title">
            <string>Functions</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTreeView" name="tvFunctions">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="autoScroll">
               <bool>true</bool>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="uniformRowHeights">
               <bool>true</bool>
              </property>
              <property name="itemsExpandable">
               <bool>false</bool>
              </property>
              <property name="expandsOnDoubleClick">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabStrings">
           <attribute name="icon">
            <iconset resource="../../../resources.qrc">
             <normaloff>:/misc_icons/res/disassemble.png</normaloff>:/misc_icons/res/disassemble.png</iconset>
           </attribute>
           <attribute name="title">
            <string>Strings</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTreeView" name="tvStrings">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="uniformRowHeights">
               <bool>true</bool>
              </property>
              <property name="itemsExpandable">
               <bool>false</bool>
              </property>
              <property name="expandsOnDoubleClick">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDataMap">
      <attribute name="title">
       <string>Data Map</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="DataMapView" name="dataMapView" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisassemblerView</class>
   <extends>QPlainTextEdit</extends>
   <header>views/formatview/disassemblerviewpage/disassemblerview/disassemblerview.h</header>
  </customwidget>
  <customwidget>
   <class>DataMapView</class>
   <extends>QWidget</extends>
   <header>views/formatview/disassemblerviewpage/datamapview/datamapview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
